tests/test_client.py

```diff
@@ -206,6 +240,37 @@ def test_get_buy_url(self):
         self.assertQueryString(url, 'something', value='else')
         self.assertQueryString(url, 'BLUB', value=[b'b1', 'b2', u'u1', 'u2'])
 
+    def test_get_subscribe_url(self):
+        item = ItemDefinition(1, 'EUR20', 'http://example.net/t', 'title', sub_id='a0_-9Z', period=12345)
+        url = self.lp.get_subscribe_url(
+            item,
+            product_key='some-product-key',
+            dialog=False,
+            use_jsevents=True,
+            transaction_reference='TX-REF',
+            consumable=True,
```

Comment:
`use_jsevents`, `transaction_reference` and `consumable` are deprecated or obsoleted. They're also not relevant to this test.